src/mmw/package.json

```diff
@@ -40,6 +40,8 @@
     "sinon": "1.14.1",
     "testem": "0.5.15",
     "turf-area": "1.1.1",
+    "turf-buffer": "^1.0.4",
+    "turf-random": "^1.0.2",
```

Comment:
Do we want to lock the version numbers? Seems like we've done that with all the other packages.